mlir/lib/Dialect/Rock/Transforms/ThreadwiseGemmLowering.cpp

```diff
@@ -89,6 +89,168 @@ struct ThreadwiseReadIntoRewritePattern
                                 ConversionPatternRewriter &b) const final;
 };
 
+//===----------------------------------------------------------------------===//
+// ThreadwiseGemmV2 lowering.
+//===----------------------------------------------------------------------===//
+struct ThreadwiseGemmv2RewritePattern
+    : public OpConversionPattern<ThreadwiseGemmOpv2> {
+  using OpConversionPattern<ThreadwiseGemmOpv2>::OpConversionPattern;
+
+  TransformMapAttr normalizeView(OpBuilder &b, Location loc,
+                                 ArrayRef<StringRef> names,
+                                 ArrayRef<int64_t> sizes, bool ignoreStartShape,
+                                 DenseSet<StringRef> ignoreNames) const {
+    unsigned pos = 0;
+    unsigned newPos = 0;
+    TopDownTMBuilder td(b, names, sizes, loc);
+    // Convert the normalizedView to a real view by ignoring
+    // the names contained in `ignoreNames` and letting the rest pass through
+    for (pos = 0; pos < names.size(); pos++) {
```

Comment:
Nit: move `unsigned pos` into the for loop.

... bigger nit: `auto [pos, name] : llvm::enumerate(names))`